### ISSUE TITLE
the caffemodel trained by your code is bigger than you release

### ISSUE THREAD

**Issue by zhangyunming** [mahyarnajibi/SSH#2]
hi, i trained the model is more than 90m, and i use the demo.py to check the caffemodel i trained,it detect nothing,what is the problem .thanks

**Comment by mahyarnajibi** [comment]
Hi,
I confirm that with the latest version of the caffe (which caffe-ssh is currently based on that) the saved models should be slightly larger. However, it should work as expected. How do you train the network? How many GPUs are you using and what is the learning rate? As mentioned in the README, the default settings are set for training on 4 GPUs. If you are using a fewer number of GPUs you should specifically lower the learning rate. Can you also check the training loss?

**Comment by zhangyunming** [comment]
thanks very much ,the caffemodel can work normally now,i have used one GPU ,batchsize is 32, interation for 100000 times and change lr at 70000 times.the loss is <0.01, not very stable.
though,i still have some questions:
1. what is the loss value finally get  when tained a good model?
2. if i do not use the pretrained model VGG16.caffemodel,trained directly,it can get a fine result?
3. if i change the base net by channel prunning to1/8. and train derectly ,it can get a small caffemodel   and good map?

sorry for the poor english. thanks for your amazing work and reply.

**Comment by xinyaoguo** [comment]
@mahyarnajibi You used 4 GPUs in your paper. Have you tried to use a GPU to train models?
I encountered a puzzled problem. I used a GPU and two GPUs to train the models. The test results are quite different. I use two pieces better than one. Do you know why?

**Comment by foralliance** [comment]
@zhangyunming HI
You point out: `batchsize is 32`, which corresponds to either BATCH_SIZE or ANCHORS_PER_BATCH


@zhangyunming 
@mahyarnajibi 
@po0ya 
HI

In the **TRAIN** of `default_config.yml`, there are two parameters about minibatch: `BATCH_SIZE` and `ANCHORS_PER_BATCH`.
Correspond to the Faster-RCNN. In my opinion, `BATCH_SIZE `corresponds to minibatch of **RCNN**, while `ANCHORS_PER_BATCH` corresponds to minibatch of **RPN**. 
In SSH,  it's just like the RPN part of Faster-RCNN, so is the parameter of `BATCH_SIZE` useless?

That's my understanding, I'm not sure it is right.
So appreciated for your reply.
